src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import './App.css';
+import React from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import VideoPlayer from './pages/VideoPlayer';
+import Login from './pages/Login';
+import AuthProvider from './providers/Auth';
+import VideosProvider from './providers/Videos';
+import Private from './components/Private';
+import Favorites from './pages/Favorites';
+
+function App() {
+  return (
+    <BrowserRouter>
```

Comment:
You should use `HashRouter` because Netlify server doesn't send the same HTML file for every path if you refresh on any page (except the home) you would get a 404.